### ISSUE TITLE
Add a state to handle properly "newly taken" and "all" pictures

### ISSUE THREAD

**Issue by stephanepechard** [PhotoBackup/client-android#20]
The "newly taken" option only sends pictures that have been taken in the last 60 seconds.
This is nice but clumsy, as it does not retry a bit later, as it was before. In case of poor connectivity or any condition that limit pictures sending, this is probably not what the user wants.

The proposal would be to introduce a new state, say "Archived", which would be set on all pictures when "newly taken" option is selected, but newly taken pictures would get the classical "Waiting" state, hence they would be sent again at retry.

The "Pictures to upload" option would toggle pictures between "Archived" and "Waiting" states.

**Comment by jkufner** [comment]
This can be easily implemented using a time constraint. Rephrase this to "Archive photos newer than X", where X is user-specified date and time (with a button to reset the option to current time).
